src/engy/app_refactor.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from .code_map import load_code_folder_to_system_prompt
+from .llm import query_llm
+from .produce_files import produce_files
+from .util import load_history, save_history
+
+
+def refactor_frontend():
+    print("Refactoring frontend into modular files", flush=True)
+    
+    # Load the current state of the project
+    system_prompt = load_code_folder_to_system_prompt('.')
+    
+    query = '''Try to refactor the "index.html", break into smaller, modular ".css", ".js", ".html" files.
```

Comment:
since this is specific to index.html and server.py. consider naming it to "split" instead of "refactor"?

Reply:
good point, let me create another PR renaming